CHANGELOG.md

```diff
@@ -17,6 +17,11 @@
   network interfaces. This address can be obtained as part of the GET
   `/vm/config`.
 
+### Changed
+
+- Changed the jailer option `--exec-file` to reject the filename without the
```

Comment:
.. to fail if the executable does not contain the string firecracker to prevent...